extensions/schema-registry/apicurio/json-schema/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,11 @@
+---
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+name: "Apicurio Registry - Json Schema"
+metadata:
+  keywords:
+  - "apicurio"
+  - "json-schema"
```

Comment:
Should we add kafka?

Reply:
Your call, but sounds reasonable to me. This is bound to the kafka use case.